internal/cli/atlas/clusters/clusters.go

```diff
@@ -97,61 +99,85 @@ func Builder() *cobra.Command {
 	return cmd
 }
 
-func NewCLILabel() atlas.Label {
+func NewCLILabel() atlasv2.NDSLabel {
 	labelValue := atlasCLILabelValue
 	if config.ToolName == config.MongoCLI {
 		labelValue = mongoCLILabelValue
 	}
 
-	return atlas.Label{
-		Key:   labelKey,
-		Value: labelValue,
+	return atlasv2.NDSLabel{
+		Key:   pointer.Get(labelKey),
+		Value: pointer.Get(labelValue),
 	}
 }
 
-func AddLabel(out *atlas.AdvancedCluster, l atlas.Label) {
+func AddLabel(out *atlasv2.ClusterDescriptionV15, l atlasv2.NDSLabel) {
 	if LabelExists(out.Labels, l) {
 		return
 	}
 
 	out.Labels = append(out.Labels, l)
 }
 
-func LabelExists(labels []atlas.Label, l atlas.Label) bool {
+func LabelExists(labels []atlasv2.NDSLabel, l atlasv2.NDSLabel) bool {
 	for _, v := range labels {
-		if v.Key == l.Key && v.Value == l.Value {
+		if v.GetKey() == l.GetKey() && v.GetValue() == l.GetValue() {
 			return true
 		}
 	}
 	return false
 }
 
-func RemoveReadOnlyAttributes(out *atlas.AdvancedCluster) {
-	out.ID = ""
-	out.CreateDate = ""
-	out.StateName = ""
-	out.MongoDBVersion = ""
+func RemoveReadOnlyAttributes(out *atlasv2.ClusterDescriptionV15) {
```

Comment:
Do you think that this can be a method that we would need to document or include in the SDK?
If yes let's create jira to cover it.